themes/codefor-theme/layouts/partials/footer.html

```diff
@@ -28,4 +28,5 @@ <h2>Spenden</h2>
 
   </div>
 </footer>
+{{ partial "analytics.html" . }}
 {{ end }}
```

Comment:
I think this would not render the analytics partial if `.Site.Params.hideFooter` is true

```suggestion
{{ end }}
{{ partial "analytics.html" . }}
```

Reply:
I can't find a reference to `hideFooter` anywhere else in the repo. Maybe we can just remove the wrapping if? Seems unnecessary to me at the moment. I also don't currently see the need for a page without a footer. Do you?

Reply:
Yep, just remove `hideFooter`

Reply:
removed it in 671693b